Sources/CoreAardvark/Logging/ARKLogObserver.h

```diff
@@ -33,4 +33,9 @@
 /// Called on a background operation queue when logs are appended to the log distributor.
 - (void)observeLogMessage:(nonnull ARKLogMessage *)logMessage;
 
+@optional
```

Comment:
oh wow I forgot this was a feature of Objective-C.

Reply:
Yep. 😄 It's great.